### ISSUE TITLE
[ECR] [Push images to repositories]: ability to push images to repositories without the need to create the sub-repo 000.dkr.ecr.eu-central-1.amazonaws.com/dev  as  ( /dev/App1 ) 

### ISSUE THREAD

**Issue by thadreez** [aws/containers-roadmap#1299]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Tell us about your request**
What do you want us to build?
NA
**Which service(s) is this request for?**
ECR  private registry  
 
**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**

The ability to push images to repositories without the need to create the sub-repo 000.dkr.ecr.eu-central-1.amazonaws.com/dev  as  ( /dev/App1 ) 

## the setup ##
- Create a repository on ECR /dev
- docker tag local/image:v1 000.dkr.ecr.eu-central-1.amazonaws.com/dev/app1:v1
- docker push 000.dkr.ecr.eu-central-1.amazonaws.com/dev/app1:v1

ERROR
name unknown: The repository with name 'dev/app1' does not exist in the registry with id '000'

**Are you currently working around this issue?**

Yes, by creating every repo, which i snot ideal 

aws ecr create-repository --repository-name dev/app1
aws ecr create-repository --repository-name dev/app2
aws ecr create-repository --repository-name dev/app3

**Additional context**
Docker registry work as intended, it will create /app1 by default 
It is not ideal to create 1000 repos for my 1000 apps

**Attachments**

**Comment by deki** [comment]
Probably already covered by #853

**Comment by dudicoco** [comment]
I agree that creating multiple repositories is unpractical and causes a management overhead, unless specifically desired to to auditing/security reasons.

As a workaround the image could be tagged with `dev:app1_v1.0.0`.
